Okta.Core.Automation/Users/EnrollOktaUserFactor.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Management.Automation;
+using Okta.Core.Models;
+
+namespace Okta.Core.Automation
+{
+    [Cmdlet("Enroll", "OktaUserFactor")]
+    public class EnrollOktaUserFactor : OktaCmdlet
+    {
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 0,
+            HelpMessage = "Id or username of the Okta user"
+        )]
+        public string IdOrLogin { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 1,
+            HelpMessage = "Name of the factor to enroll the user with. Use one of the following string: okta_question, okta_sms, okta_otp (Okta Verify), okta_push (Okta Verify Push), google_otp (Google Authenticator), symantec_vip, rsa_token, duo, yubikey_token, okta_call (voice call)"
```

Comment:
It looks like some of the newer factors such as U2F and Windows Hello are missing from this list